instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
+  spec.add_dependency 'opentelemetry-instrumentation-amazing_gem', '~> 0.10.0'
```

Comment:
This doesn't match the `VERSION` in `version.rb`.

Reply:
Fixed.